atomic-reactor.spec

```diff
@@ -340,6 +341,7 @@ cp -a docs/manpage/atomic-reactor.1 %{buildroot}%{_mandir}/man1/
 %{python2_sitelib}/atomic_reactor/cli
 %{python2_sitelib}/atomic_reactor/plugins
 %{python2_sitelib}/atomic_reactor/schemas
+%{python2_sitelib}/atomic_reactor/utils
 %exclude %{python2_sitelib}/atomic_reactor/koji_util.py*
```

Comment:
As the patches to move the util modules are merged, these util lines should also be removed from the spec file